src/pfe/file-watcher/server/src/controllers/projectEventsController.ts

```diff
@@ -78,6 +78,8 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
         let isSettingFileChanged = false;
         let isProjectBuildRequired = false;
 
+        const projectHandler = await projectExtensions.getProjectHandler(projectInfo);
+        const builtByExtension = projectHandler.builtByExtension;
```

Comment:
Could we skip the `try` block below entirely, then `isProjectBuildRequired` will never be set to `true`

Reply:
We need to find out if the `.cw-settings` file has been changed, and need to proceed project settings change.